miso-web/src/main/webapp/scripts/list_servicerecord.js

```diff
@@ -113,8 +116,14 @@ ListTarget.servicerecord = {
             });
             list += "</ul>";
             return list;
+          } else if (type === "filter") {
```

Comment:
It turned out that mRender was called three times in some cases, 
one with type === "display", type === "filter", and type === "type"
we handled "display" one above, but did not for "filter" and "type"
so I added an extra if statement.

Reply:
Now you're returning a string for "filter" and `null` for "type." They should be the same. Was there a reason for this change?

Reply:
I could make it return the string for both cases. I thought it should only return string for filter and not for type.